### ISSUE TITLE
Prepare folder structure for future coding

### ISSUE THREAD

**Issue by BraveSpirit** [LadislavSzolik/simple-steps#1]
I would suggest using best practise from here: [https://scotch.io/tutorials/angularjs-best-practices-directory-structure](url)

**Comment by LadislavSzolik** [comment]
Okey, I think my first angular project I did that based on this site. Did you get time to check typescript?

**Comment by BraveSpirit** [comment]
Yep, I had a chance today to look deeper into the typescript. There are two things:
1) I really wanna try it out, especially because of compile-time checks and all the nice stuff it provides
2) I am a bit scared, as JS is still a standard (e.g. at our company) and using typescript makes you go away from the usual JS.

We could give it a try.

**Comment by LadislavSzolik** [comment]
Why we do this project? 
- Train and improve our skills 
- Learn something new
- Build a useful tool -> here we could say than let's use known technologies, however the other two points are more important.
- Find a way towards self employment - this point is for me. I know I can't be self employed from nothing and I know what we learn in work does not prepare us to run something alone. 

... some other goal?

I think it make sense if the development environment is defined by you, I will adapt and focus more on the features and usability...as I mentioned. Choose whatever you want to use and let's kick it, we adapt as we go. Is it ok for you? 

Let's make a new version by end of August. ;-)

**Comment by BraveSpirit** [comment]
The last point is also a point for me. And you are very right, that just working is not helping to prepare :)

One of the goals can be to make money out of the result, but this should not be an ultimate goal though. If we do something awesome and useful, it will mean we can profit a bit as well.
I am ok with what you wrote and let's just start and adapt things if needed during the work :)

May be we could set a milestone with all the functionality we would like to have for the next draft version.

**Comment by BraveSpirit** [comment]
I made it. I cannot commit though. I guess I am not included in collaborators in the settings of the project.
